app/models/audit/event/policy.rb

```diff
@@ -75,7 +75,7 @@ def facility
       private
 
       def user
-        @user || @policy_version.role
+        @user ||= @policy_version.role
```

Comment:
why? can you explain?

Reply:
@uCatu , If I understand correctly `@policy_version.role` is a DB call which might be costly. So, after doing it once I want to save the value into `@user` to save that call next time